test/cli-tests/test-bitswap.js

```diff
@@ -12,7 +12,7 @@ const createTempNode = require('../utils/temp-node')
 const repoPath = require('./index').repoPath
 
 describe('bitswap', function () {
-  this.timeout(20000)
+  this.timeout(80000)
```

Comment:
I could break this into another PR, but timeouts were making my local tests fail. :(